x/airdrop/keeper/claim.go

```diff
@@ -26,6 +26,17 @@ func (k Keeper) ClaimDaily(ctx sdk.Context, airdropId, claimer string) error {
 			userAllocation.ClaimType.String())
 	}
 
+	// Confirm we're not past the decision date
+	currentTime := ctx.BlockTime().Unix()
+	if currentTime > airdrop.ClaimTypeDeadlineDate.Unix() {
+		return types.ErrAfterDecisionDeadline
+	}
```

Comment:
i don't think we should do this for the daily check

Reply:
oh :facepalm:

Reply:
fixed 1ef4a2f0b